website/docs/downloads/development-builds.md

```diff
@@ -90,10 +90,11 @@ function set_ci_status(workflow_file, os_name, description, page = 1) {
 }
 
 document.addEventListener("DOMContentLoaded", () => {
-    set_ci_status("linux.yml", "linux", "Linux");
-    set_ci_status("macos.yml", "macos", "macOS");
-    set_ci_status("windows-msys2.yml", "msys2", "Windows MSYS2 builds ¹");
-    set_ci_status("windows-msvc.yml", "windows", "Windows MSVC builds ²");
+    set_ci_status("linux.yml", "linux", "Linux x86-64");
```

Comment:
That detail is fine for the Linux release, I think.

Reply:
x86_64, with an underscore, not hyphen.